mathics/builtin/box/expression.py

```diff
@@ -198,15 +198,16 @@ def get_option_values(self, elements, **options):
         evaluation = options.get("evaluation", None)
         if evaluation:
             default = evaluation.definitions.get_options(self.get_name()).copy()
-            options = ListExpression(*elements).get_option_values(evaluation)
-            default.update(options)
         else:
             from mathics.core.parser import parse_builtin_rule
 
             default = {}
             for option, value in self.options.items():
                 option = ensure_context(option)
                 default[option] = parse_builtin_rule(value)
+
```

Comment:
This change is needed to make some of the low-level tests work, especially when an `Evaluation` object is not available.